devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -101,9 +101,14 @@ def parse(String description) {
 			map = parseIasMessage(description)
 		} else {
 			Map descMap = zigbee.parseDescriptionAsMap(description)
-			if (descMap?.clusterInt == 0x0001 && descMap.commandInt != 0x07 && descMap?.value) {
+
+			if (descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.commandInt != 0x07 && descMap.value) {
```

Comment:
might as well change the `?.` to just `.` for the `clusterInt` as well to be consistent.  `descMap` can't be null anyways.  Same goes for the other DTHs

Reply:
I'm not sure I understand what you mean. I am going to keep descMap?.clusterInt as the first item in the if (since we do it in every other if in these DTHes), just to at least be safe. Unless it adds some ungodly overhead. This way, if descMap for some reason is null we don't spit up exceptions. (I do see some other areas where it is used from previous commits that isn't needed. And now I am tempted to go on a cleaning spree... 😮 )

Reply:
parseDescriptionAsMap will always return an empty map, unless it throws an exception itself.  But it's not a big deal.  It can stay as is.